.github/workflows/release.yml

```
@@ -10,26 +10,21 @@ on:
   workflow_dispatch:
 
 jobs:
-  goreleaser:
+  release:
     runs-on: ubuntu-latest
-    timeout-minutes: 30
+    timeout-minutes: 10
+
     steps:
-      - name: Checkout
+      - name: checkout
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
-      - name: Set up Go
+      - name: setup dependencies
         uses: actions/setup-go@v4
         with:
           cache-dependency-path: ${{env.WORKING_DIRECTORY}}/go.sum
           go-version-file: ${{env.WORKING_DIRECTORY}}/go.mod
```

```suggestion
          cache-dependency-path: go.sum
          go-version-file: go.mod
```

and remove the working_directory env, you don't need it
we use it an ak since we have multiple libraries in one repo

---

removed